src/garage/tf/algos/batch_polopt.py

```diff
@@ -138,6 +139,18 @@ def process_samples(self, itr, paths):
 
         max_path_length = self.max_path_length
 
+        if self.flatten_input:
+            paths = list(
```

Comment:
This would definitely be clearer as a list comprehension.

Reply:
@krzentner fixed